Source/WebCore/page/writing-tools/WritingToolsController.mm

```diff
@@ -267,8 +283,10 @@
         state->replacementLocationOffset += static_cast<int>(suggestion.replacement.length()) - static_cast<int>(suggestion.originalRange.length);
     }
 
-    if (finished)
+    if (finished) {
```

Comment:
should this be done in `didEnd` instead of here?

Reply:
> For proofreading, suppression ends once the last suggestion is received